server/etcdserver/version/version_test.go

```diff
@@ -20,61 +20,109 @@ import (
 
 	"github.com/coreos/go-semver/semver"
 	"github.com/stretchr/testify/assert"
-	"go.etcd.io/etcd/api/v3/version"
 	"go.etcd.io/etcd/raft/v3/raftpb"
 	"go.etcd.io/etcd/server/v3/mvcc/backend"
 	betesting "go.etcd.io/etcd/server/v3/mvcc/backend/testing"
 	"go.etcd.io/etcd/server/v3/mvcc/buckets"
 	"go.uber.org/zap"
 )
 
+var (
+	V3_4 = semver.Version{Major: 3, Minor: 4}
+	V3_7 = semver.Version{Major: 3, Minor: 7}
+)
+
 func TestUpdateStorageVersion(t *testing.T) {
```

Comment:
Does this test still belong in the version test. As we don't really call anything from that file? Just a thought.

Reply:
Forgot to update the test name, it should be `TestMigrate`. It tests the `Migrate` function from `version.go`, so it should stay here. However I plan to move it to `storage/schema/schema.go` when https://github.com/etcd-io/etcd/pull/13198 is merged.

Reply:
Nice, sgtm 👍